.github/workflows/master.yaml

```diff
@@ -15,8 +15,8 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
-      - name: Unshallow
-        run: git fetch --prune --unshallow
+      - name: Fetch tags
+        run: git fetch --depth=1 origin '+refs/tags/*:refs/tags/*'
```

Comment:
Can we just leave this like it was so it matches the [documentation it came](https://goreleaser.com/actions/) from?

Reply:
Was torn on switching to tags or adding in `fetch-depth: 0` to save a step.

```
    # Number of commits to fetch. 0 indicates all history.
    # Default: 1
    fetch-depth: '
```

Reply:
You are right, getting the tags is better then `fetch-depth: 0`. I think "new Git" lets you do `git fetch --tags --depth=1` instead of spelling out the refspec.